src/org/ods/orchestration/usecase/JiraUseCase.groovy

```diff
@@ -299,6 +299,16 @@ class JiraUseCase {
         }
     }
 
+    private boolean mustRun(testIssue) {
+        return !(project.buildParams?.targetEnvironmentToken == 'P' &&
```

Comment:
@jafarre-viewnext it may be worth factoring this into a method

Reply:
Just committed a change. Is this what you had in mind?